packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -613,10 +728,11 @@ class _BottomNavigationBarState extends State<BottomNavigationBar> with TickerPr
     assert(debugCheckHasMaterialLocalizations(context));
```

Comment:
While you're here, could you add a `debugCheckHasMediaQuery`?

Reply:
Done.